### PR TITLE
Remove Bower.

### PR DESCRIPTION
**Pull request by camskene**
PR for #86.

### CODE REVIEW

File: config/ember-try.js [hunk 1]
```diff
@@ -3,49 +3,49 @@ module.exports = {
   scenarios: [
     {
       name: 'ember-1.13',
-      bower: {
-        dependencies: {
+      npm: {
+        devDependencies: {
           'ember': '~1.13.0'
```
Comment (trentmwillis) [inline config/ember-try.js]: I think these all need to be updated to `ember-source`.

**Comment by rwjblue** [comment]
FYI - we have to use bower in ember-try config to test ember < 2.11 (ember-source was not published prior)...

**Comment by camskene** [comment]
Not sure why CI is sad here.

**Comment by MelSumner** [comment]
Closing based on length of time open with no progress. Please feel free to work on it and resubmit PR - thank you!

**Pull request closed without merging**